tritium-tracing/src/main/java/com/palantir/tritium/tracing/JavaTracingTracer.java

```diff
@@ -21,7 +21,7 @@
 
     @Override
     public void startSpan(String operationName) {
-        com.palantir.tracing.Tracer.startSpan(operationName);
+        com.palantir.tracing.Tracer.fastStartSpan(operationName);
```

Comment:
We'll need to ensure that downstream consumers all pickup tracing-java 2.16.0+ at the same time as tritium bump. I'm on the fence about providing a temporary fallback:

```
    private volatile NoSuchMethodError failure = null;

    @Override
    public void startSpan(String operationName) {
        try {
            if (failure == null) {
                com.palantir.tracing.Tracer.fastStartSpan(operationName);
            } else {
                fallbackToSlowStartSpan(operationName, failure);
            }
        } catch (NoSuchMethodError e) {
            fallbackToSlowStartSpan(operationName, e);
        }
    }

    private void fallbackToSlowStartSpan(String operationName, NoSuchMethodError e) {
        // temporary bridge in case downstream consumer has
        com.palantir.tracing.Tracer.startSpan(operationName);
        if (failure == null) {
            failure = e;
            log.error("Falling back to slower Tracer.startSpan because tracing-java is not version 2.16.0+", e);
        }
    }
```

Reply:
Aside from one large internal project, I expect most consumers are using https://github.com/palantir/gradle-consistent-versions to ensure proper dependencies.

Reply:
There is at least one other major project not using GCV that we'll need to coordinate excavation.

Reply:
why don't they use it? can i add GCV to them ?

Reply:
The larger of the two products is already on the upgraded tracing-java version, and the smaller does not depend directly on tracing, so it will pull the highest version from its dependencies (or fail resolution until a human does so).

I'm more worried about testing correctness of the fallback approach in tritium than upgrading products which attempt to run unsupported configurations. It's common for other libraries to bump dependency versions in order to take advantage of new features.

Reply:
@schlosna GCV should mean you no longer need to worry about consumers of forcing stuff to too low versions.  I think it'll work seamlessly without the fallback.

Reply:
@iamdanfox yeah GCV "should" do the work, just want to make sure someone can ensure it all flows through and can shepherd things through so we don't see failures as I probably won't have cycles to do that for many of those projects